openapi.yml

```diff
@@ -10257,4 +10285,3 @@ servers:
       default: discourse.example.com
 components:
   schemas: {}
-...
```

Comment:
What's that? 🙂

Reply:
Looks like it is literally 3 dots that were added by mistake a previous PR:

https://github.com/discourse/discourse_api_docs/commit/6ff476d2a41e61bca04aa42bb51730a16263a9e1#diff-5016bdc926b6c0f649f363739d09159a1db664aae19a08fa6b38b03406f040dcR10260